client/src/components/PageLayout/index.tsx

```diff
@@ -5,13 +5,31 @@ import {
   AlertDescription,
   AlertTitle,
 } from '@chakra-ui/react';
-import React from 'react';
+import React, { useEffect } from 'react';
 import { SkipNavContent } from '@chakra-ui/skip-nav';
 
+import Link from 'next/link';
+import { useCalendarIntegrationStatusLazyQuery } from '../../generated/graphql';
+import { useAuth } from '../../modules/auth/store';
+import { checkPermission } from '../../util/check-permission';
+import { Permission } from '../../../../common/permissions';
 import { Header } from './Header';
 import { Footer } from './component/Footer';
 
 const PageLayout = ({ children }: { children: React.ReactNode }) => {
+  const { user } = useAuth();
+  const canAuthenticateWithGoogle = checkPermission(
+    user,
+    Permission.GoogleAuthenticate,
+  );
+  const [loadStatus, { data }] = useCalendarIntegrationStatusLazyQuery();
+
+  useEffect(() => {
+    if (canAuthenticateWithGoogle) {
+      loadStatus();
+    }
+  }, [canAuthenticateWithGoogle]);
```

Comment:
```suggestion
  const { data } = useCalendarIntegrationStatusQuery({
    skip: !canAuthenticateWithGoogle,
  });
```
This seems like the expected way to conditionally load.  Same basic idea as `useEffect`, but it's more concise and declarative.

Reply:
That's pretty handy!